tools/ci_build/github/azure-pipelines/orttraining-linux-gpu-ortmodule-test-ci-pipeline.yml

```diff
@@ -0,0 +1,63 @@
+trigger: none
```

Comment:
After getting into master, we should merge this with orttraining-linux-gpu pipeline.

Reply:
I will create a task for this. We may need to increase number of VMs for it